litmus-portal/build_docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t maya-app ./backend
+docker build -t maya-frontend ./frontend
```

Comment:
This can be part of Makefile

Reply:
Create a common Makefile for both frontend and backend

Reply:
I'm so sorry Raj, I do not have enough expertise for these things related to Makefile. This code was not written by me.